ena-manifest.yaml

```diff
@@ -4,13 +4,13 @@
 ---
 metadata:
   schemaVersion: 2.0.0
-  release: 1.3.4
+  release: 1.3.5-dev
```

Comment:
manifest version needs to be a non dev version in order to be uploaded for use in deployments